packages/moonstone/Button/Button.module.less

```diff
@@ -67,26 +67,41 @@
 		animation-timing-function: ease-out;
 	}
 
-	.client {
-		padding: @moon-button-border-width;	// We match the amount removed by the line-height above, so the text doesn't overlap the border
+	.clientContainer {
+		padding: 0 @moon-button-h-padding;
 
-		// Colored under-bar
+		// Colored dot for remote buttons
 		&::before {
-			content: "";
+			content: "O";
```

Comment:
I'm not sure if it's because I don't have fonts installed correctly, but I see differences in the width of the dot depending on what locale I'm setting.  With en-US, it isn't round, but with something like ar-SA it is.

Also not sure if it is a problem, but I wanted to get comments going.

Reply:
I'm skeptical that setting this rule is the direction we should be taking with this.